base/applications/regedit/treeview.c

```diff
@@ -441,41 +446,42 @@ static BOOL InitTreeViewImageLists(HWND hwndTV)
     HICON hico;       /* handle to icon  */
     INT cx = GetSystemMetrics(SM_CXSMICON);
     INT cy = GetSystemMetrics(SM_CYSMICON);
+    HMODULE hShell32 = GetModuleHandleW(L"SHELL32");
```

Comment:
upper, or lowercase? with `.dll` extension ?

Reply:
Seems to work fine without extension, but ok I have added it and made it lowercase.